### DISCUSSION TITLE
Count system idle time

### DISCUSSION THREAD

**Discussion by ivanlee1007** [timmo001/system-bridge#1195] [Q&A] [answered]
Is it possible to count system_idle_time like IOTLink do so I can decide when to shutdown my computer?

**Comment by timmo001** [comment, marked as answer]
This is now in the system module of 3.x.x

**Comment by aderusha** [comment]
Where exactly would one find this?  Just deployed 3.11.2 and I'm not finding any idle counters available under System (or anywhere else).
